### PR TITLE
created model for a driver, dao for a driver, services for a driver

### PR DESCRIPTION
**Pull request by BusyhinD**
and a manufacturer

### CODE REVIEW

File: src/main/java/mate/jdbc/Main.java [hunk 1]
```diff
@@ -1,7 +1,48 @@
...
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //Test operation with driver
+        Driver driver = new Driver();
```
Comment (nacenik) [inline src/main/java/mate/jdbc/Main.java:14]: Better to give a more understandable names for variables. Fix it in all places 
```suggestion
        Driver bob = new Driver();
```

File: src/main/java/mate/jdbc/dao/DriverDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            List<Driver> drivers = new ArrayList<>();
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                drivers.add(getDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get list of drivers"
+                    + " from drivers table.", e);
```
Comment (nacenik) [inline src/main/java/mate/jdbc/dao/DriverDaoImpl.java:68]:
```suggestion
            throw new DataProcessingException("Couldn't get list of drivers from table.", e);
```

File: src/main/java/mate/jdbc/service/DriverServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        return driverDao.get(id);
```
Comment (nacenik) [inline src/main/java/mate/jdbc/service/DriverServiceImpl.java:22]: better to add `orElseThrow()` and throw NoSuchElementException with a message. Same for manufacture

**Review by nacenik — CHANGES_REQUESTED** [review]

**Review by okuzan — APPROVED** [review]
Good work :)

**Pull request closed without merging**